02-Logging/00-index.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
<p>Not a big deal either way, but do we want to call these element of the HOOMD-blue API systems or subsystems?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Good point, we shouldn't use the word <code>system</code>  for two different concepts.</p>